modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Iri.scala

```diff
@@ -108,6 +108,9 @@ object Iri {
   final def apply(string: String): Either[String, Iri] =
     urn(string) orElse url(string) orElse relative(string)
 
+  @SuppressWarnings(Array("EitherGet"))
+  private[rdf] final def unsafe(string: String): Iri = apply(string).right.get
```

Comment:
unless we're defining a collection of function definitions inherited from a parent type, let's define the implementation on the next line.